test/jdk/tools/jpackage/share/ConcurrentTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2018, 2020, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.util.Date;
+import jdk.jpackage.test.TKit;
+import jdk.jpackage.test.PackageType;
+import jdk.jpackage.test.JPackageCommand;
+import jdk.jpackage.test.Annotations.Test;
+
+/**
+ * Concurrent test.  Using ToolProvider, run several jpackage test concurrently
+ */
+
+/*
+ * @test
+ * @summary Concurrent jpackage command runs using ToolProvider
+ * @library ../helpers
+ * @key jpackagePlatformPackage
+ * @build jdk.jpackage.test.*
+ * @modules jdk.jpackage/jdk.jpackage.internal
+ * @compile ConcurrentTest.java
+ * @run main/othervm/timeout=360 -Xmx512m jdk.jpackage.test.Main
+ *  --jpt-run=ConcurrentTest
+ */
+public class ConcurrentTest {
+
+    @Test
+    public static void test() {
+
+        final JPackageCommand cmd1 =
+                JPackageCommand.helloAppImage("com.other/com.other.Hello")
+        .useToolProvider(true)
+        .setPackageType(PackageType.getDefault())
```

Comment:
`.removeArgumentWithValue("--type")` can be used to make jpackage create default native package. This will eliminate need to introduce `packageType.getDefault()` method.

However, I'd rather replace JPackageCommand with PackageTest in this test scenario. Something like this:
```
final PackageTest cmd1 = new PackageTest()
.configureHelloApp()
.addInitializer(cmd -> {
    cmd.setArgumentValue("--name", "ConcurrentOtherInstaller");
});
```

`cmd1.run(PackageTest.Action.CREATE);` would result in a package creation but will no unpack or install will be attempted.

PackageTest is better as it will run tests on created package making sure it is valid.